metabase/metabase
# Allow Box props in Icon

Make `Icon` work closer to other `metabase/ui` components and allow to pass `Box` props. The only downside is that we cannot filter `styled`  properties automatically this way - so we should migrate from `styled(Icon)<SomeProps>`.

How to verify:
- Find any usage of `Icon`. Try `c="text-dark"` or `m="lg"` and see if it works.
